### PR TITLE
Language edits and correct typos

### PR DESCRIPTION
**Pull request by veshivas**
Also updated some section titles to use title case instead
of sentence case.

### CODE REVIEW

File: docs/manual/gammaray-qresource-browser.qdoc [hunk 1]
```diff
@@ -27,16 +27,16 @@
 
     The resource browser view allows to inspect the content of assets compiled into the application binary using the QResource system.
     The tree view on the left side shows the file and folder hierarchy, the content view on the right allows to inspect the file content
-    if it is a supported text or image format.
+    if it is a supported text or borderedimage format.
```
Comment (vkrause) [inline docs/manual/gammaray-qresource-browser.qdoc]: search/replace mistake?

File: docs/manual/gammaray-action-inspector.qdoc [hunk 2]
```diff
@@ -26,12 +26,12 @@
     \section1 Overview
 
     The action inspector shows all QAction instances that exist in the target application. Information about
-    the display name, checkable states and shortcuts and shortcut priorities are displayed.
+    the display name, checkable states, and shortcuts with their priorities are displayed.
 
-    \image gammaray-action-inspector.png
+    \borderedborderedimage gammaray-action-inspector.png
```
Comment (vkrause) [inline docs/manual/gammaray-action-inspector.qdoc]: double search/replace? occurs in a few places below as well

**Comment by vkrause** [comment]
Tried to integrate this now, but I'm getting "warning: Unknown command '\borderedimage'" for each instance of \borderedimage (with Qt 5.7), when building offline docs, and no images in the output anymore.

**Comment by veshivas** [comment]
The "\borderedimage" macro was added in 9f45d2ab03a643348c6f0c464aa48755b301498d. Looks like your 5.7 build is missing that. The quickest way to fix this is to patch the macros.qdocconf, which you'll find under the QT_INSTALL_DOCS path. Run 'qmake -query' to know this path, and append "macro.borderedimage       = "\\div {class=\"border\"} \\image \1\n\\enddiv"" to macros.qdocconf.

**Comment by vkrause** [comment]
Same problem with Qt 5.8 for both offline and online docs, and I've verified the macro is defined there. Weird. Anyway, as GammaRay supports Qt versions all the way back to 4.8 I need to define this macro locally anyway for compatibility, and then it seems to work fine. I'll integrate your patch as is then, thanks!

**Pull request merged**